utils/python/CIME/XML/component.py

```diff
@@ -22,7 +22,31 @@ def get_value(self, name, attribute={}, resolved=False):
             compnode = self.get_node("components")
             comps = self.get_nodes("comp", root=compnode)
             for comp in comps:
-                components.append(comp.text())
+                components.append(comp.text)
             return components
+        elif name == "help":
+            rootnode = self.get_node("help")
+            helptext = rootnode.text
+            return helptext
+        elif name == "description":
+            rootnode = self.get_node("description")
+            compsets = {}
+            descs = self.get_nodes("desc", root=rootnode)
+            for desc in descs:
+                attrib = desc.attrib["compset"]
+                text = desc.text
```

Comment:
I think my may is clearer - so I'll keep it that way for now. We agreed on this.
Done